recipes/systemd_service.rb

```diff
@@ -69,5 +69,6 @@ class ::Chef::Recipe
       'AccuracySec' => "#{node['chef_client']['splay']}sec",
     }
   )
-  action(timer ? [:create, :enable, :start] : [:disable, :delete])
+  action(timer ? [:create, :enable, :start] : [:stop, :disable, :delete])
+  notifies :restart, to_s, :delayed
```

Comment:
`self` is implicit, so the to_s call in this context returns `"systemd_unit[chef client.timer]"`